pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[build-system]
+requires = [
+  "setuptools",
+  "wheel"
+]
+build-backend = "setuptools.build_meta"
```

Comment:
If you use [setuptools-scm](https://github.com/pypa/setuptools_scm/), you can set the version automatically using git tags

(but the current approach seems totally fine and reasonable!)

Reply:
Thanks - maybe in the future we'll switch over.